frontend/views/containers/user-settings/AppLogs.vue

```diff
@@ -118,10 +137,16 @@ export default ({
     ]),
     addLog (entry: Object) {
       if (entry) {
+        if (this.form.source === 'browser' && entry.source !== 'browser') return
+        if (this.form.source === 'serviceworker' && entry.source !== 'sw') return
```

Comment:
I don't quite get this logic... shouldn't we want to keep logs, and not return, if `entry.source` is `combined`?